docs/storage/index.md

```diff
@@ -72,9 +69,12 @@ The use of storage is billed according to the [billing policy](../runjobs/lumi_e
     billing of the project scratch and project flash as detailed on the
     [billing page][billing].
 
-    The project flash space is only meant for very short term file storage.
+    It is recommended to only keep the data on /flash during the time when it's actively needed.
     
 
+Notice that we have no backups for any data in the LUMI storage system. Users should pay attention to move their important data elsewhere, and take regular backups. One option for a backup solution is to copy your data also to LUMI-O.
```

Comment:
Notice that there are **no** backup for any of the storage systems of LUMI.

Users should pay attention to make secure copy of important data elsewhere,